dts/bindings/sensor/invensense,icm4268x-common.yaml

```diff
@@ -0,0 +1,368 @@
+      accel-aaf-bw = <ICM4268X_DT_ACCEL_AAF_42HZ>;
+      gyro-aaf-bw = <ICM4268X_DT_GYRO_AAF_42HZ>;
+    };
+
+    Where,
+    ICM4268X_DT_ACCEL_ODR_2000 represents 2kHz as rate of sample capture by the
+    accelerometer.
+
+    ICM4268X_DT_GYRO_ODR_2000 represents 2kHz as rate of sample capture by the
+    gyroscope.
+
+    ICM4268X_DT_ACCEL_AAF_42HZ represents 42kHz as the bandwidth of the
+    Anti Alias Filter for the Accelerometer device.
+
+    ICM4268X_DT_GYRO_AAF_42HZ represents 42kHz as the bandwidth of the
+    Anti Alias Filter for the Gyroscope device.
+
+compatible: "invensense,icm4268x-common"
+include: [sensor-device.yaml, spi-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      This property describes the GPIO specification for INT line from ICM4268x package
+      on to your device. The property values should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  int1-pol:
+    type: int
+    default: 0
+    description: |
+      Choose if the interrupt1 polarity will be Active High or Ative Low.
+      Default polarity is active low.
+
+      0 # ICM4268X_DT_INT1_POLARITY_ACTIVE_LOW
+      1 # ICM4268X_DT_INT1_POLARITY_ACTIVE_HIGH
+
+    enum: [0, 1]
+
+  int1-mode:
+    type: int
+    default: 0
+    description: |
+      Choose if the interrupt will be automatically latched and then cleared (pulsed mode)
+      Or latch and clear happens only when user code requests the data (latched mode).
+      Default interrupt mode is pulsed.
+
+      0 # ICM4268X_DT_INT1_MODE_PULSED
+      1 # ICM4268X_DT_INT1_MODE_LATCHED
+
+    enum: [0, 1]
+
+  fifo-mode:
+    type: int
+    default: 0
+    description: |
+      Select what to do when FIFO is full. Stream to FIFO mode will overwrite LRU frame.
+      Stop on Full mode will prevent writing any frame if both FIFO and cache are full.
+      Default fifo mode is bypass.
+
+      0 # ICM4268X_DT_FIFO_MODE_BYPASS
+      1 # ICM4268X_DT_FIFO_MODE_STREAM
+      3 # ICM4268X_DT_FIFO_MODE_STOP_ON_FULL
+
+    enum: [0, 1, 3]
+
+  fifo-ths-int-clr:
+    type: int
+    default: 1
+    description: |
+      Fifo threshold Interrupt Clearing Options.
+      Default fifo threshold interrupt (in latched mode) gets cleared on status bit read.
+
+      1 # ICM4268X_DT_FIFO_THS_INT_CLR_STSRD
+      2 # ICM4268X_DT_FIFO_THS_INT_CLR_DATARD
+      3 # ICM4268X_DT_FIFO_THS_INT_CLR_BOTH
+
+    enum: [1, 2, 3]
+
+  fifo-full-int-clr:
+    type: int
+    default: 1
+    description: |
+      Fifo Full Interrupt Clearing Options.
+      Default fifo full interrupt (in latched mode) gets cleared on status bit read.
+
+      1 # ICM4268X_DT_FIFO_FULL_INT_CLR_STSRD
+      2 # ICM4268X_DT_FIFO_FULL_INT_CLR_DATARD
+      3 # ICM4268X_DT_FIFO_FULL_INT_CLR_BOTH
+
+    enum: [1, 2, 3]
+
+  tmst-en:
+    type: boolean
+    description: |
+      Enable or disable the timestamp register.
+      Timestamp register is enabled by default.
```

Comment:
The default of a boolean is false....

Reply:
Changed the parameter name to tmst-dis instead.. please check